R/create.R

```diff
@@ -289,7 +289,7 @@ create_from_github <- function(repo_spec,
   rstudio <- rstudio %||% rstudio_available()
   rstudio <- rstudio && !is_rstudio_project(proj_get())
   if (rstudio) {
-    use_rstudio()
+    use_rstudio(reformat = FALSE)
```

Comment:
```suggestion
    use_rstudio(reformat = !fork)
```

There is part of me that thinks we should only set reformat = FALSE in a fork, but I can also ignore that part of me.

Reply:
Regardless of whether you're forking or not, I feel like you should leave the code alone.